test/node-unit/utils.spec.js

```diff
@@ -45,5 +45,16 @@ describe('utils', function () {
         );
       });
     });
+    describe('isNumeric()', function () {
```

Comment:
[Tests] I tried removing the `isFloat()` from `isNumeric`, making it just `return !isNan(input)`. These three unit tests still passed but `npm run test-node` had a failure:

```plaintext
Error: Option  is unsupported by the mocha cli
    at createUnsupportedError (/Users/josh/repos/mocha/lib/errors.js:124:101)
    at /Users/josh/repos/mocha/lib/cli/options.js:52:523
    at Array.reduce (<anonymous>)
    at parse (/Users/josh/repos/mocha/lib/cli/options.js:52:186)
    at loadOptions (/Users/josh/repos/mocha/lib/cli/options.js:85:354)
    at Object.<anonymous> (/Users/josh/repos/mocha/bin/mocha.js:9:674)
    at Module._compile (node:internal/modules/cjs/loader:1467:14)
    at Module.replacementCompile (/Users/josh/repos/mocha/node_modules/append-transform/index.js:60:13)
    at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
    at Object.<anonymous> (/Users/josh/repos/mocha/node_modules/append-transform/index.js:64:4) {
  code: 'ERR_MOCHA_UNSUPPORTED'
}
```

So, there must be some case missing from tests. Requesting that be tested too please.

Reply:
Thanks for the catch. Done!